### PR TITLE
Auto recover

### PR DESCRIPTION
**Pull request by ghalym**
Adding the FIX to GCC State change

### CODE REVIEW

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU [hunk 1]
```diff
@@ -54,15 +52,18 @@ END_IF
...
-IF (rV <=9.6 ) AND (rV>=0.6) THEN
+IF (rV <=9.6 ) AND (rV>=0.6) AND (IG.xHV_SW) THEN
 	IG.eState := Valid; // normal
-ELSIF rV >= 0.18 AND rV <= 0.22 THEN 
+ELSIF rV >= 0.18 AND rV <= 0.22 AND (IG.xHV_SW) THEN 
 	IG.eState := ValidLo; //LO
-ELSIF rV > 9.6 AND rV<= 9.9 THEN
+ELSIF rV > 9.6 AND rV<= 9.9 AND (IG.xHV_SW) THEN
 	IG.eState := ValidHi; //HIGH
 ELSIF rV < 0.18 THEN //
```
Comment (klauer) [inline L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU]: Probably a dumb question as I'm drive-by reviewing for a project I have nothing to do with, but should this include: `AND Ig.xHV_SW` like the others?
(If so, it might be a bit more readable to do a top-level `IF xHV_SW THEN (check voltages) ELSE set_off_state`)

**Review by klauer — COMMENTED** [review]
Looks like `LineIDs.dbg` snuck into the repository in this PR.

(Feel free to ignore my sideline comments - was just looking this over during the eng gen.)

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU (hunk 1 above)
Comment (jyin999) [reply L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU]: is it missing the range rv< 0.6 and rv>0.22? which should be valid or valid low?

Comment (slacAWallace) [reply L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU]: Should these values be local constants? Avoiding hardcoded values?

**Review by jyin999 — APPROVED** [review]
